src/message/headers/StreamMessageHeader.js

```diff
@@ -17,7 +17,6 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     flexDirection: 'row',
     backgroundColor: '#eee',
-    overflow: 'hidden',
```

Comment:
We can't do this or the scroll view performance will degrade significantly. Ever child view of scrollview must have 'overflow: hidden' set for the removeClippedSubviews optimization